Lunch/LunchPFTableViewController.swift

```diff
@@ -0,0 +1,138 @@
+        let delete = UITableViewRowAction(style: .Normal, title: "Delete") { action, index in
+            self.objectAtIndexPath(indexPath)?.deleteInBackgroundWithBlock({ (success: Bool, error: NSError?) -> Void in
+                if success && error == nil {
+                    self.loadObjects()
+                    self.tableView.reloadData()
+                }
+            })
+
+        }
+        delete.backgroundColor = UIColor.redColor()
+
+        // VOTE
+        let vote = UITableViewRowAction(style: .Normal, title: "Vote") { action, index in
+            var voteCount = self.objects?[indexPath.row][self.voteKey] as! Int
+            self.objectAtIndexPath(indexPath)?.incrementKey("vote")
+            self.objectAtIndexPath(indexPath)?.saveInBackgroundWithBlock({ (success: Bool, error: NSError?) in
+                if success && error == nil {
+                    self.loadObjects()
+                    self.tableView.reloadData()
+                }
+            })
+        }
+        vote.backgroundColor = UIColor.blueColor()
+        return [vote, delete]
+    }
+
+    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        // the cells you would like the actions to appear needs to be editable
+        return true
+    }
+
+    override func tableView(tableView: UITableView, commitEditingStyle editingStyle: UITableViewCellEditingStyle, forRowAtIndexPath indexPath: NSIndexPath) {
+        // you need to implement this method too or you can't swipe to display the actions
+    }
+
+    @IBAction func addPlaceBarItemTapped(sender: UIBarButtonItem) {
+        var inputTextField: UITextField?
+        let eateryPrompt = UIAlertController(title: "Enter an Eatery", message: "Look up the best place to eat!", preferredStyle: UIAlertControllerStyle.Alert)
+        eateryPrompt.addAction(UIAlertAction(title: "Cancel", style: UIAlertActionStyle.Default, handler: nil))
+        eateryPrompt.addAction(UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler: { (action) in
+            self.newEateryName = inputTextField?.text
+            let eatery = PFObject(className: self.classKey)
+            eatery[self.placeKey] = self.newEateryName
+            eatery[self.voteKey] = 0
+            eatery.saveInBackgroundWithBlock({ (success: Bool, error: NSError?) in
+                if success && error == nil {
+                    self.loadObjects()
+                    self.tableView.reloadData()
+                }
+            })
+        }))
+        eateryPrompt.addTextFieldWithConfigurationHandler({(textField: UITextField!) in
+            textField.placeholder = "Eatery"
+            inputTextField = textField
+        })
+        presentViewController(eateryPrompt, animated: true, completion: nil)
+    }
+
+    @IBAction func logoutBarItemTapped(sender: UIBarButtonItem) {
+        PFUser.logOut()
```

Comment:
Why aren't you using the [asynchronous](https://parse.com/docs/osx/api/Classes/PFUser.html#//api/name/logOutInBackgroundWithBlock:) version?